cmd/juju/controller/destroy.go

```diff
@@ -674,7 +674,7 @@ func (c *destroyCommandBase) getControllerEnvironFromStore(
 func (c *destroyCommandBase) getControllerEnvironFromAPI(
 	api destroyControllerAPI,
 	controllerModelConfigAPI modelConfigAPI,
-) (environs.Environ, error) {
+) (environs.BootstrapEnviron, error) {
```

Comment:
Considering this is used on destroyCommandBase, does it need a BootstrapEnviron? Maybe just Environ does make "more sense" semantically

Reply:
Yes ill named in the presence of k8s.